docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -1746,7 +1748,7 @@ Result:
 Returns the count of the specified `unit` boundaries crossed between the `startdate` and the `enddate`.
 The difference is calculated using relative units, e.g. the difference between `2021-12-29` and `2022-01-01` is 3 days for unit `day` (see [toRelativeDayNum](#torelativedaynum)), 1 month for unit `month` (see [toRelativeMonthNum](#torelativemonthnum)) and 1 year for unit `year` (see [toRelativeYearNum](#torelativeyearnum)).
 
-If unit `week` was specified, `date\_diff` assumes that weeks start on Monday. Note that this behavior is different from that of function `toWeek()` in which weeks start by default on Sunday.
+If unit `week` was specified, `date\_diff` assumes that weeks start on Monday by default. You can change this behavior with [`first_day_of_week`](/docs/en/operations/settings/settings.md/#first-day-of-week)
```

Comment:
Should we also document function `age`?

Reply:
AFAICS `age` doesn't depend on first day of week since everything is relative. Also, I haven't applied any change to `age` code branch in `dateDiff.cpp`